public/jsmodules/querycountry.js

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
The `querycity.js` and `querycountry.js` files both deal with hitting the `/search-*` endpoints, and can reasonably live in the same file